azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeExitEvent.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * Event with caused throwable.
+ */
+public final class CallCompositeExitEvent {
+    private final CallCompositeErrorEvent errorEvent;
+
+    /**
+     * Create {@link CallCompositeExitEvent} with error event.
+     *
+     * @param errorEvent error event.
+     */
+    public CallCompositeExitEvent(final CallCompositeErrorEvent errorEvent) {
```

Comment:
It is weird that we have CallCompositeErrorEvent inside of another event. Would it be better to have an object or enum like ExitCause - that would be an user initiated, API (exit() function call) and error. 
If it is an error cause, then we populate Throwable and error code.

Should we also have a no parameters constructor?